manage

```diff
@@ -275,6 +275,17 @@ elif [ "$1" == 'install' ] || [ "$1" == 'uninstall' ];then
     mv "$logfile" "$(echo "$logfile" | sed 's+-incomplete-+-success-+g')" #rename logfile to indicate it was successful
     
   else #if app failed to install/uninstall
+
+    #remove dummy deb if app failed to install (to avoid dummy debs being left on a users install for broken apps)
+    if [[ ${mode} == "install" ]]; then
```

Comment:
This also needs to check if the app is a `standard` app, not a `package` app.

Reply:
alright should be good now